composer.json

```diff
@@ -16,15 +16,15 @@
         }
     ],
     "require": {
-        "php": ">=5.3.2",
-        "symfony/framework-bundle": "~2.3||~3.0",
-        "symfony/serializer": "~2.3||~3.0",
-        "symfony/console": "~2.3||~3.0",
+        "php": "^5.3.9|^7.0",
```

Comment:
not sure if we still want to support versions below 5.6 .. but if its no extra effort, I guess its fine.

Reply:
if it works, let's keep it and once it causes an issue, one can increase the min version?

Reply:
👍

Reply:
I think it is in line with the rest of our dependencies, so it makes sense to only increase this if we want to use features present in PHP >5.3.